src/telegram-types.ts

```diff
@@ -140,8 +185,9 @@ export type ExtraVideo = MakeExtra<'sendVideo', 'video'>
 export type ExtraVideoNote = MakeExtra<'sendVideoNote', 'video_note'>
 export type ExtraVoice = MakeExtra<'sendVoice', 'voice'>
 
-export type IncomingMessage = TT.Message
+export type IncomingMessage = Message
 
+/** @deprecated */
```

Comment:
This is a partial redefinition of the arguments of `sendInvoice` in `typegram`. Should this be removed?